src/main/java/io/bitrise/trace/step/InjectTraceTask.java

```diff
@@ -574,4 +646,46 @@ static int getSmallestNonNegativeNumber(final int... numbers) {
         return Arrays.stream(numbers).filter(i -> i >= 0).min().orElse(-1);
     }
     //endregion
+
+    /**
+     * Inner data class for a range value (which has a start and an end).
```

Comment:
nit: might be nice to explain where this gets used ?

Reply:
I am not sure it is the proper thing to do, because as soon as we use it in a different place too, we have to update it.  "Find usages" will show this information to the developer.

Reply:
Hmm. Okay. that's fair!